src/action/Button/Button.tsx

```diff
@@ -39,77 +39,76 @@ const buttonVariant = (variant: Variant = "primary") => {
 };
 
 export type Size = "small" | "normal" | "large";
-const buttonSize = (size: Size = "normal") => {
+const buttonSize = (size: Size = "normal", hasChildren: boolean = true) => {
   switch (size) {
     case "small":
       return css`
         gap: ${sizing(6)};
-        height: ${sizing(20)};
-        padding: 0px ${sizing(14)};
+        ${hasChildren && `padding: 0px ${sizing(14)};`}
```

Comment:
I think it's necessary to keep the padding for the button with children AND the button with only icon.
The icon button is naturally smaller than the button with text, but if you remove the padding when there is only an icon, the button is really small.
So I think is better to have padding fot icon button too

Reply:
good point, I'll fix it